sdk/monitor/monitor-query/src/models/publicMetricsModels.ts

```diff
@@ -100,7 +102,7 @@ export interface MetricsQueryResult {
   /** The integer value representing the cost of the query, for data case. */
   cost?: number;
   /** The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested. */
-  timespan: string;
+  timespan: TimeInterval;
```

Comment:
nit: not sure if `timespan` is a known industry term or if it should `timeSpan`

Reply:
https://github.com/Azure/azure-sdk-for-js/issues/17545

Reply:
timespan is one word, so not updating that